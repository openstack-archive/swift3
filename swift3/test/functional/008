#!/bin/bash

# DELETE Object

. ./common

S3USER=tester

_s3_put /bucket

echo data > $TEST_DIR/data
MD5=$(_md5 $TEST_DIR/data) CONTENT_TYPE='application/octet-stream' \
    _s3_put /bucket/object -T $TEST_DIR/data \
    -H 'Content-Disposition: attachment; filename="a.txt.gz"' \
    -H 'Content-Encoding: gzip' \
    -H "Content-Length: $(_file_size $TEST_DIR/data)" \
    -H 'Expect: 100-continue' \
    -H 'x-amz-storage-class: STANDARD' -D - | _filter_curl

_s3_delete /bucket/object

rm $TEST_DIR/data

