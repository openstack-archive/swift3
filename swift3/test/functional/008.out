QA output created by 008
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester --contentMd5 YTfN5Ik8WfdvAFqBI9jo5g== --contentType application/octet-stream -- -X PUT -T /TEST_DIR/data -H Content-Disposition: attachment; filename="a.txt.gz" -H Content-Encoding: gzip -H Content-Length: 5 -H Expect: 100-continue -H x-amz-storage-class: STANDARD -D - http://SWIFT_HOST/bucket/object... 200
HTTP/1.1 100 Continue

HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
ETag: "ETAG"
Last-Modified: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

> s3curl --id tester -- -X DELETE http://SWIFT_HOST/bucket/object... 204
